src/Provider/ApiServiceProvider.php

```diff
@@ -65,7 +65,7 @@ public function register()
             'Dingo\Api\Console\Command\Docs',
         ]);
 
-        if (class_exists('Illuminate\Foundation\Application', false)) {
+        if (class_exists('Illuminate\Contracts\Foundation\Application', false)) {
```

Comment:
That's exactly why we're checking that the `Illuminate\Contracts\Foundation\Application` interface which [Laravel implements](https://github.com/laravel/framework/blob/5.2/src/Illuminate/Foundation/Application.php) exists and not for the `Illuminate\Contracts\Container\Container` interface which [Lumen solely implements](https://github.com/laravel/lumen-framework/blob/5.2/src/Application.php).